primer entregable

fgarcia1105 opened the pull request:
Cambios realizados
1. Se crean las ramas main-fernando-garcia, dev-fernando-garcia y task-fernando-garcia
2. Se modifica el modelo de datos Movie y se agrega Results
3. Se modifica el consumo del servicio 
4. Se agregan controles de acceso

Commit:
Clonacion del proyecto y generacion de ramas

Commit:
- Se crean las Ramas: main-feriado-garcia, dev-fernando-garcia y task-fernando-garcia
- Se modifica el model de datos Movie y se agrega el modelo Results
- Se modifica el consumo de la API
- Se aplican controles de acceso a clases, métodos y propiedades

JosueQuinonesRivera commented:
This is pointing to the right branch. Please check the comments on the first PR. Those apply to these changes

Commit:
-  Se quitaron comentarios de prueba
	-  Se reemplaza print por debugPrint
	-  Se agrega método para obtener datos en la vista

Commit:
Merge branch 'dev-fernando-garcia'
